packages/suite/src/views/firmware/index.tsx

```diff
@@ -63,19 +69,18 @@ export const Firmware = ({ shouldSwitchFirmwareType }: FirmwareProps) => {
         resetReducer();
     };
 
-    const [cachedDevice, setCachedDevice] = useState<TrezorDevice | undefined>(device);
-
     // some of the application states can be reused here.
     // some don't make sense handling here as they are handled somewhere up the tree
     // some must be handled in lower layers because of specifics of fw update process (eg. device-disconnected)
     const getSuiteApplicationState = () => {
-        if (!device) return;
+        if (!device) return null;
+        return null;
         // device features cannot be read, device is probably used in another window
-        if (device.type === 'unacquired') return DeviceAcquire;
+        // if (device.type === 'unacquired') return DeviceAcquire;
         // Webusb unreadable device (HID)
-        if (device.type === 'unreadable') return DeviceUnreadable;
+        // if (device.type === 'unreadable') return DeviceUnreadable;
         // device features unknown (this shouldn't happened tho)
-        if (!device.features) return DeviceUnknown;
+        // if (!device.features) return DeviceUnknown;
```

Comment:
Why are these commented out?

Reply:
not sure, just some testing. whatever changes I made to packages/suite* are not set in stone definitely